components/QuizCardPage/QuizForm/index.js

```diff
@@ -14,48 +15,57 @@ export default function QuizForm() {
-  const question = questions[index] ? questions[index].question : "";
-  const currentQuestion = questions[index];
+  const [index, setIndex] = useState();
 
+  useEffect(() => {
+    const randomIndex = Math.floor(Math.random() * (questions.length - 1));
+    setIndex(randomIndex);
+  }, []);
   return (
     <>
       <Header points1={points1} points2={points2}></Header>
       <StyledForm>
-        <QuizCard questions={question}></QuizCard>
-        <Timeline
-          isTimerPaused={isTimerPaused}
-          setIsDisabled={setIsDisabled}
-          timeLeft={timeLeft}
-          setTimeLeft={setTimeLeft}
-        />
-        <AnswerButtons
-          points1={points1}
-          setPoints1={setPoints1}
-          selectedAnswer={selectedAnswer}
-          setSelectedAnswer={setSelectedAnswer}
-          isDisabled={isDisabled}
-          setIsDisabled={setIsDisabled}
-          setIsTimerPaused={setIsTimerPaused}
-          timeLeft={timeLeft}
-          index={index}
-          currentQuestion={currentQuestion}
-        ></AnswerButtons>
-        <NextButton
-          isDisabled={isDisabled}
-          setSelectedAnswer={setSelectedAnswer}
-          index={index}
-          setIndex={setIndex}
-          setIsDisabled={setIsDisabled}
-          setIsTimerPaused={setIsTimerPaused}
-          setTimeLeft={setTimeLeft}
-          setClickCounter={setClickCounter}
-          clickCounter={clickCounter}
-          points1={points1}
-          setPoints1={setPoints1}
-          currentQuestion={currentQuestion}
-        ></NextButton>
+        {index && (
+          <>
+            <QuizCard
+              key={questions[index].id}
+              question={questions[index].question}
+            ></QuizCard>
+            <Timeline
+              isTimerPaused={isTimerPaused}
+              setIsDisabled={setIsDisabled}
+              timeLeft={timeLeft}
+              setTimeLeft={setTimeLeft}
+            />
+            <AnswerButtons
+              points1={points1}
+              setPoints1={setPoints1}
+              selectedAnswer={selectedAnswer}
+              setSelectedAnswer={setSelectedAnswer}
+              isDisabled={isDisabled}
+              setIsDisabled={setIsDisabled}
+              setIsTimerPaused={setIsTimerPaused}
+              timeLeft={timeLeft}
+              index={index}
+              currentQuestion={questions[index]}
+            ></AnswerButtons>
+            <NextButton
+              isDisabled={isDisabled}
+              setSelectedAnswer={setSelectedAnswer}
+              index={index}
+              setIndex={setIndex}
+              setIsDisabled={setIsDisabled}
+              setIsTimerPaused={setIsTimerPaused}
+              setTimeLeft={setTimeLeft}
+              setClickCounter={setClickCounter}
+              clickCounter={clickCounter}
+              points1={points1}
+              setPoints1={setPoints1}
+              currentQuestion={questions[index]}
+            ></NextButton>
```

Comment:
If you are not using `children` as prop for a component you don't need the closing tag but can make it self-closing 🙂